presto-docs/src/main/sphinx/release/release-0.202.rst

```diff
@@ -0,0 +1,78 @@
+* Fix query failure in aggregation spilling
+  (`issue #10587 <https://github.com/prestodb/presto/issues/10587>`_).
+* Remove support for ``SHOW PARTITIONS`` statement.
+* Improve support for correlated subqueries containing equality predicates.
+* Improve performance of correlated ``EXISTS`` subqueries.
+* Limit the number of grouping sets in a ``GROUP BY`` clause.
+  The default limit is ``2048`` and can be set via the ``analyzer.max-grouping-sets``
+  configuration property, and ``max_grouping_sets`` session property.
+* Allow coercion between row types regardless of field names.
+  Previously, a row type is coercible to another only if the field name in the source type
+  matches the target type, or when target type has anonymous field name.
+* Increase default value for ``experimental.filter-and-project-min-output-page-size`` to 500kB.
+* Optimize array equals operator for ``bigint`` and ``double`` types.
+* Respect ``X-Forwarded-Proto`` header in client protocol responses.
+* Add support for column-level access control.
+  Connectors have not yet been updated to take advantage of this support.
+* Add support for correlated subqueries with correlated ``OR`` predicates.
+* Add :func:`multimap_from_entries` function.
+* Add :func:`bing_tiles_around` function.
+* Add :func:`ST_NumGeometries`, :func:`ST_GeometryN` and :func:`ST_ConvexHull` geospatial functions.
+* Add :func:`wilson_interval_lower` and :func:`wilson_interval_upper` functions.
+* Add ``IS DISTINCT FROM`` for ``JSON`` and ``IPADDRESS`` type.
+
+
+Hive Changes
+------------
+
+* Fix optimized ORC Writer encoding of ``TIMESTAMP`` before ``1970-01-01``.
+* Fix query failure when a Hive bucket has no splits. This commonly happens when a
+  predicate filters some buckets out entirely.
+* Remove the ``hive.bucket-writing`` config property.
+* Add support for creating and writing bucketed sorted tables. The list of
+  sorting columns may be specified using the ``sorted_by`` table property.
+  Writing to sorted tables can be disabled using the ``hive.sorted-writing``
+  config property or the ``sorted_writing_enabled`` session property. The
+  maximum number of temporary files for can be controlled using the
+  ``hive.max-sort-files-per-bucket`` property.
+* Collect and store basic table statistics (``rowCount``, ``fileCount``, ``rawDataSize``, ``totalSize`)
+  on ``INSERT`` and ``CREATE TABLE AS SELECT``
+* Add ``hive.orc.tiny-stripe-threshold`` config property and ``orc_tiny_stripe_threshold`` session
+  property to control the stripe/file size threshold when ORC reader decides to read multiple
+  consecutive stripes or entire fires at once. Previously, this feature piggybacks on other properties.
+
+Thrift Connector Changes
+------------------------
+
+* Allow Thrift connector to provide header to service.
+
+CLI Changes
+-----------
+
+* Add peak memory usage to ``--debug`` output.
+* Add support for setting trace token via the ``TraceToken`` client info property. TraceToken is passed
```

Comment:
This note needs some work. Mention the command line argument. Also, `ConnectorSession` has zero relevance to users of the CLI.